client/controllers/events/eventFiltration.coffee

```diff
@@ -23,6 +43,12 @@ Template.eventFiltration.onCreated ->
       filters.status =
         $in: status
 
+    selectedLocations = @selectedLocations.find().map (location) ->
+      location.name
+    if selectedLocations.length
+      locationProjection = "locations.#{@countryLevel.get()}"
+      filters[locationProjection] = $in: selectedLocations
```

Comment:
@jgoley The geonames don't have information we can use to determine containment for continent and region names like "West Africa," so I think it will be difficult to do. I'll put something in the backlog for it.